opensrp-reveal/src/main/java/org/smartregister/reveal/view/ListTasksActivity.java

```diff
@@ -385,12 +386,21 @@ public void startJsonForm(JSONObject form) {
 
     @Override
     public void displaySelectedFeature(Feature feature, LatLng point) {
+        adjustFocusPoint(point);
         kujakuMapView.centerMap(point, ANIMATE_TO_LOCATION_DURATION, mMapboxMap.getCameraPosition().zoom);
         if (selectedGeoJsonSource != null) {
             selectedGeoJsonSource.setGeoJson(FeatureCollection.fromFeature(feature));
         }
     }
 
+    private void adjustFocusPoint(LatLng point) {
+        final double VERTICAL_OFFSET = -0.0003;
```

Comment:
This offset works well in southern hemisphere but in northern hemisphere it renders the point beneath the card view.

Check if clicked point is in northern hemisphere and set the offset to be positive 0.0003

Reply:
yes